### PR TITLE
Restructured so that method name specifies display

### PR DESCRIPTION
**Pull request by ChrisDobby**
Restructured so that type of display for the series is specified by the method name and the data type is overloaded as the first parameter

### CODE REVIEW

**Comment by LukeAllen** [comment]
I just tried it out; looks great!

**Comment by LukeAllen** [comment]
@tpetricek It looks like I don't have write access to this new repository. Could you add me as a collaborator in the repository settings?

File: docs/content/tutorial.fsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ The access to gnuplot is managed through the `GnuPlot` type. When creating an in
 can specify the path to the `gnuplot` executable. If you do not pass a path explicitly, then
 FnuPlot assumes that you have gnuplot in your `PATH` variable:
 *)
-let gp_default = new GnuPlot()
+//let gp_default = new GnuPlot()
```
Comment (tpetricek) [inline docs/content/tutorial.fsx]: I think the docs look nicer if this is not commented out.

**Comment by tpetricek** [comment]
@ChrisDobby Nice work - the new API looks really great! If you could remove the comment from the docs, that would be nice (but it's not a big deal). 

@LukeAllen I added you as a collaborator (that is, I added @fsprojects/fnuplot-owners and added you there). Have you used the ProjectScaffold release process before? Maybe we could try it on @ChrisDobby's update :-). There is a [documentation on releasing things here](http://fsprojects.github.io/ProjectScaffold/release-process.html). 

Though I personally don't use the release script as described there and run `build Release` (which asks you for a GitHub password a few times to publish documentation etc.) and then manually push the nuget package (`nuget push bin/FnuPlot.X.Y.Z.nupkg`). The release script is probably simpler though!

**Comment by ChrisDobby** [comment]
Uncommented the line as I meant to do before I committed it in the first place!

**Comment by tpetricek** [comment]
Thanks - merged!

@LukeAllen Let me know if you want to try doing the release (we'll also need to add a new version info to `RELEASE_NOTES.md`). I'm happy to do this, but it would be good to check if it works on your machine & you have all the necessary rights!

**Comment by LukeAllen** [comment]
@tpetricek Yeah, I'll read the docs and make sure I understand the NuGet release process, and I'll try to do the release myself over the next couple days.

**Pull request merged**